src/stores/messages/MessagesStore.ts

```diff
@@ -149,6 +149,18 @@ export default class MessagesStore {
 		}
 	};
 
+	@action
+	public detailify = (messageId: string) => {
```

Comment:
those are not real words, can we use something more reasonable, and `toggleMessageBeautify` and `toggleMessageDetailedRaw` aren't used anywhere